erpnext/regional/report/irs_1099/irs_1099.py

```diff
@@ -16,9 +16,14 @@
 
 def execute(filters=None):
 	filters = filters if isinstance(filters, _dict) else _dict(filters)
+	company = frappe.db.get_default("company")
 	if not filters:
 		filters.setdefault('fiscal_year', get_fiscal_year(nowdate())[0])
-		filters.setdefault('company', frappe.db.get_default("company"))
+		filters.setdefault('company', company)
+
+	region = frappe.db.get_value("Company", fieldname = ["country"], filters = { "name": company })
```

Comment:
If I change the company in the filters or don't have a default company set, this will still break. Check the region of the company in the filter currently

Reply:
Okay